### PR TITLE
Seed file

### PR DESCRIPTION
**Pull request by ViaMarcus**
Creates
5 users, user0@mail.com ... user4. password0 and so on.
50 requests randomly between these 5 users
75 offers, randomly between these requests.
These offers may have random status, and the request status is reflected to show that if needed.
200 messages randomly between the offers, with random sender.
Screen Shot 2020-06-25 at 16 00 50

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -51,10 +51,10 @@
     t.datetime "updated_at", precision: 6, null: false
     t.integer "reward"
     t.integer "status", default: 0
-    t.integer "category", default: 0
     t.bigint "helper_id"
-    t.float "long"
-    t.float "lat"
+    t.integer "category", default: 0
+    t.float "long", null: false
+    t.float "lat", null: false
```
Comment (ViaMarcus) [inline db/schema.rb:57]: This is because I never migrated my development db when adding these I think, only the test db

**Review by designerofthing — APPROVED** [review]
impressive

**Review by erikbjoern — APPROVED** [review]
Wow.

This was quick.

**Review by kermit-klein — APPROVED** [review]

**Comment by kermit-klein** [comment]
Good job!

**Pull request merged**